java/kotlin-extractor/src/main/kotlin/KotlinFileExtractor.kt

```diff
@@ -94,6 +95,9 @@ open class KotlinFileExtractor(
         if (d.isFakeOverride) {
             return true
         }
+        if ((d as? IrFunction)?.descriptor?.isHiddenToOvercomeSignatureClash == true) {
+            return true
+        }
```

Comment:
Can we add an example to such a hidden function in the tests?

Reply:
I actually don't know how to deliberately manufacture one yet, but if you revert this change you should see a wide variety of tests fail with Kotlin 1.7.0 because of `CharBuffer.get`